### DISCUSSION TITLE
Anyone know how to upgrade from rustdesk-1.1.9-0.x86_64 on Fedora to 1.2.1?

### DISCUSSION THREAD

**Discussion by andy-page-developer** [rustdesk/rustdesk#5042] [Q&A] [answered]
Is there a way to manually update or do we need to wait until the repo's have updated?

**Comment by SergeyMy** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/releases/tag/nightly 
`sudo dnf install https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.1-0.x86_64.rpm`

**Reply by andy-page-developer** [reply]
Thank you , that worked great.
